astro/src/content/quickstarts/quickstart-wordpress-web.mdx

```diff
@@ -0,0 +1,337 @@
+
+Back in <Breadcrumb label="Pages"/>, for both the account and change pages, click <Uielement label="Edit"/>, enable <Uielement label="For Logged-In Users Only"/> at the bottom right, and click <Uielement label="Update"/>. Now if you log out and browse to either page, you won't be able to view it.
+
+<img src="/img/quickstarts/quickstart-wordpress-web/for-logged-in.png" alt="Page settings"/>
+
+### Send User To Account After Log In
+
+The final improvement you can make to the login process is to send the user directly to their account page instead of the dashboard after login.
+
+Search for and install the plugin "Remove Dashboard Access" by TrustedLogin.
+
+Browse to <Breadcrumb label="Settings -> Dashboard Access"/>. Change the value of <Field label="Redirect URL"/> to `http://localhost:3000/account`. Click <Uielement label="Save Changes"/>.
+
+<img src="/img/quickstarts/quickstart-wordpress-web/dash-access.png" alt="Page settings"/>
+
+Now if you log out and log in as `richard@example.com`, who is not an administrator, you'll be redirected to the account page. Log in again as `admin@example.com` to continue editing the site.
+
+## Customization
+
+You now have everything you need to configure FusionAuth authentication on your site. To complete this demonstration application, you'll add some content to the three blank pages. There are many ways to create custom pages on {frontmatter.technology}, including:
+
+- Editing the theme.
+- Making a child theme and editing that.
+- Adding new template files for each page.
+- Making custom shortcodes.
+- Making custom blocks.
+- Making custom plugins.
+
+The easiest way to demonstrate the ChangeBank sample pages is to paste custom content into a plugin, but note that is not the most maintainable solution for a real site. If your site is large or complex, rather take the time to study custom themes and create your own.
+
+For now, search for and add the plugin "WP Coder" by Wow-Company.
+
+### Add An Image
+
+Browse to <Breadcrumb label="Media"/> in the sidebar. Click <Uielement label="Add New"/> and select `money.jpg` from the `complete-application` directory.
+
+<img src="/img/quickstarts/quickstart-wordpress-web/media.png" alt="Add image"/>
+
+Click on the image once it has been uploaded. Copy the "File URL" to use later. It should look like `http://localhost:3000/wp-content/uploads/2023/09/money.jpg`.
+
+### Add Custom Page Shortcodes
+
+You'll now create three shortcodes with the HTML, CSS, and JavaScript for each page.
+
+Browse to <Breadcrumb label="Wow Plugins -> WP Coder"/>. Click <Uielement label="Add New"/>. In the title box at the top, enter `home`. In the <Field label="HTML code"/> box, copy the file contents from `complete-application/home.html`. In the `<img src` at the bottom of this code, enter the file URL for the money image you uploaded. Click the <Breadcrumb label="CSS Code"/> tab, and copy the file contents from `changebank.css`. Click <Uielement label="Save Changes"/>.
+
```

Comment:
Please import the files so it is easy to copy paste.

Reply:
code imported